index.bs

```diff
@@ -751,14 +751,16 @@ A <dfn id=concept-sensor>sensor</dfn> has an associated [=ordered set|set=]
 of <dfn>activated sensor objects</dfn>.
 This set is initially [=set/is empty|empty=].
 
-A [=sensor=] has an associated <dfn>latest reading</dfn> [=ordered map|map=]
-which holds the latest available [=sensor readings=].
+For the current [=browsing context=] a [=sensor=] has an associated <dfn>latest reading</dfn>
```

Comment:
The current browsing context's sensor has

Reply:
done